app/scripts/services/projects.js

```diff
@@ -23,6 +23,7 @@ angular.module('openshiftConsole')
           return  AuthService
                     .withUser()
                     .then(function() {
+                      AuthorizationService.getProjectRules(projectName);
```

Comment:
I'm trying to decide, I hate to wait for the rules to load before fetching other stuff but I'm not sure there is a better way to guarantee we get the digest loop triggered off this data loading.